armhf.Dockerfile

```diff
@@ -22,15 +22,12 @@ ENV NVM_DIR /tmp/.nvm
 RUN mkdir -p $NVM_DIR \
  && curl -s https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash \
  && . $NVM_DIR/nvm.sh \
- && nvm install 10 \
- && nvm install 12 \
- && nvm install 13 \
- && nvm install 14 \
+ && nvm install 16 \
  && chmod a+rwX -R $NVM_DIR
 
 # Ensure a new enough version of CMake is available.
-RUN cd /opt \
-    && curl -O -J https://cmake.org/files/v3.15/cmake-3.15.2-Linux-x86_64.tar.gz \
-    && tar zxf cmake-3.15.2-Linux-x86_64.tar.gz
-
-ENV PATH "/opt/cmake-3.15.2-Linux-x86_64/bin:$PATH"
\ No newline at end of file
+RUN mkdir -p /home/jenkins/cmake && \
+    cd /home/jenkins/cmake && \
+    curl -O -J https://cmake.org/files/v3.23/cmake-3.23.2-linux-x86_64.tar.gz &&\
+    tar zxf cmake-3.23.2-linux-x86_64.tar.gz
```

Comment:
Is there a lower bound to the version of `cmake` we should be using?
If so, we should maby assert that in the build-android.js script?

Reply:
It is handled by the root `CMakeList.txt` (`cmake_minimum_required`) but I can see there is a mismatch.